pkg/apis/console/v1alpha1/types.go

```diff
@@ -15,6 +15,17 @@ const (
 	defaultVersion   = "latest"
 )
 
+type ManagementState string
```

Comment:
Why is this duped instead of imported from `openshift/api`?

Reply:
Because its just a const.  I can fix that, however.